src/quill-editor.component.ts

```diff
@@ -178,11 +179,11 @@ export class QuillEditorComponent implements AfterViewInit, ControlValueAccessor
     }
 
     let err: {
-      minLengthError?: {given: number, minLength: number};
-      maxLengthError?: {given: number, maxLength: number};
-      requiredError?: {empty: boolean}
+      minLengthError?: { given: number, minLength: number };
+      maxLengthError?: { given: number, maxLength: number };
+      requiredError?: { empty: boolean }
     } = {},
-    valid = true;
+      valid = true;
```

Comment:
why that space? if you want to indent this --> indent the whole block even the err object